diff -r 02852230c305 src/Makefile.am
--- a/iperf3/src/Makefile.am	Fri Jan 10 13:22:11 2014 -0800
+++ b/iperf3/src/Makefile.am	Tue Jan 21 15:39:14 2014 -0500
@@ -8,6 +8,7 @@
 libiperf_a_SOURCES      = \
                         cjson.c \
                         cjson.h \
+                        flowlabel.h \
                         iperf.h \
                         iperf_api.c \
                         iperf_api.h \
