diff -r 02852230c305 src/iperf_api.c
--- a/iperf3/src/iperf_api.c	Fri Jan 10 13:22:11 2014 -0800
+++ b/iperf3/src/iperf_api.c	Tue Jan 21 15:31:02 2014 -0500
@@ -533,7 +533,7 @@
         {"file", required_argument, NULL, 'F'},
         {"affinity", required_argument, NULL, 'A'},
         {"title", required_argument, NULL, 'T'},
-#if defined(linux)
+#if defined(linux) && defined(TCP_CONGESTION)
         {"linux-congestion", required_argument, NULL, 'C'},
 #endif
         {"help", no_argument, NULL, 'h'},
@@ -734,7 +734,7 @@
 		client_flag = 1;
                 break;
 	    case 'C':
-#if defined(linux)
+#if defined(linux) && defined(TCP_CONGESTION)
 		test->congestion = strdup(optarg);
 		client_flag = 1;
 #else /* linux */
diff -r 02852230c305 src/iperf_tcp.c
--- a/iperf3/src/iperf_tcp.c	Fri Jan 10 13:22:11 2014 -0800
+++ b/iperf3/src/iperf_tcp.c	Tue Jan 21 15:31:02 2014 -0500
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2009-2011, The Regents of the University of California,
+ * Copyright (c) 2009-2014, The Regents of the University of California,
  * through Lawrence Berkeley National Laboratory (subject to receipt of any
  * required approvals from the U.S. Dept. of Energy).  All rights reserved.
  *
@@ -184,7 +184,7 @@
                 return -1;
             }
         }
-#if defined(linux)
+#if defined(linux) && defined(TCP_CONGESTION)
 	if (test->congestion) {
 	    if (setsockopt(s, IPPROTO_TCP, TCP_CONGESTION, test->congestion, strlen(test->congestion)) < 0) {
 		close(s);
@@ -373,7 +373,7 @@
     }
 #endif
 
-#if defined(linux)
+#if defined(linux) && defined(TCP_CONGESTION)
     if (test->congestion) {
 	if (setsockopt(s, IPPROTO_TCP, TCP_CONGESTION, test->congestion, strlen(test->congestion)) < 0) {
 	    close(s);
